book/src/background/fields.md

```diff
@@ -3,7 +3,7 @@
 A fundamental component of many cryptographic protocols is the algebraic structure known
 as a [field]. Fields are sets of objects (usually numbers) with two associated binary
 operators $+$ and $\times$ such that various [field axioms][field-axioms] hold. The real
-numbers $\mathbb{R}$ are an example of a field with an uncountably infinite number of
+numbers $\mathbb{R}$ are an example of a field with an uncountable number of
```

Comment:
```suggestion
numbers $\mathbb{R}$ are an example of a field with an uncountably infinite number of
```
We do mean to say "uncountably infinite" here: https://mathworld.wolfram.com/UncountablyInfinite.html

Reply:
Yep, this was just a small nit for readability, as uncountability implies the set is infinite. 

I can change it back if you think the previous reading is clearer or better terminology wise.